Library/Formula/libstfl.rb

```diff
@@ -18,6 +18,10 @@ class Libstfl < Formula
   patch :DATA
 
   def install
+    if Formula["ruby"].installed?
```

Comment:
Well, I'm wondering two things. One, when we don't add the path directly (as in this PR) and we don't add it indirectly (by adding the dependency), how is it finding out about the brew ruby at all? And two, why is that when we make this change
```
-    args = ["CC=#{ENV.cc} -pthread", "prefix=#{prefix}"]
+    args = ["prefix=#{prefix}"]
```
that the build succeeds?